docs/Upgrading.md

```diff
@@ -25,6 +25,13 @@ Now install that version of `react-native` in your project with `npm install --s
 ```sh
 $ npm install --save react-native@X.Y 
 # where X.Y is the semantic version you are upgrading to
+npm WARN peerDependencies The peer dependency react@~R included from react-native...
+```
+
+If you saw a warning about the peerDependency, also install `react`:
```

Comment:
@ncuillery Interesting point. If npm 3 prints a warning about say react@15.4.1, and npm2 fails with an error about react@15.4.1, I would **always** want to run 'npm install --save react@15.4.1, because my version of React Native requires it, wouldn't it?

When would I want to run a different command than 'npm install --save react@15.4.1'?